fury/utils.py

```diff
@@ -855,6 +855,35 @@ def rotate(actor, rotation=(90, 1, 0, 0)):
         prop3D.SetOrientation(newTransform.GetOrientation())
 
 
+def one_chanel_to_vtk(data):
```

Comment:
missing a `n` in channel

Reply:
Also, I think this function should be in the `convert` module

Reply:
No sure I understand this function. Why do you mean by one channel? it seems you expect only a 2D image so there is only 1 channel. it would make more sense if you were expected a 3D image and the index of the channel that you want. Can you clarify ?

Reply:
Can you answer the questions above